### DISCUSSION TITLE
No boxes available inside Custom Plugin

### DISCUSSION THREAD

**Discussion by lorenzoantei** [OFFLINE-GmbH/oc-boxes-support#104] [Help] [answered]
Hello. I'm trying to create boxes for my plugin. I can use them correctly within the boxes/editorcontroller. If I try within my plugin, I get: "There are no boxes available" .

![Screenshot from 2024-02-07 15-26-38](https://github.com/OFFLINE-GmbH/oc-boxes-support/assets/58749463/2a23201e-d4cd-42ba-8963-9888611a0921)
![Screenshot from 2024-02-07 15-32-00](https://github.com/OFFLINE-GmbH/oc-boxes-support/assets/58749463/92891f7e-2066-4138-bfe8-b8a9a31c9e6c)

**Comment by tobias-kuendig** [comment, marked as answer]
@lorenzoantei do you have a `partialContexts` defined in your `fields.yaml` or is the value unset?

**Comment by lorenzoantei** [comment]
Right. I forgot to remove it from example.
